packages/core/src/index.ts

```diff
@@ -27,6 +27,7 @@ export type BuildSendMailOptions<T> = {
   transport: Transporter<T>;
   defaultFrom: string;
   configPath: string;
+  processHtml?: (html: string) => string;
```

Comment:
@psugihara should this follow the same `options` api you did in the render or do you think this is fine?

btw changing the name was the right call. i had similar thoughts so i'm glad you thought to do this

Reply:
I think this is fine because it's already in an options object. My reason for adding that was just in case we want to add more options in the future without breaking users who might be relying on the current type signature for `render`.